lib/index.js

```diff
@@ -29,10 +29,17 @@ export default function customElementToReact (elementClass, options = {}) {
   const skipProps = customProps.slice() // Keep a copy
   const tagName = `${dashCase}-${options.suffix || 'react'}`.replace(/\W+/g, '-').toLowerCase()
 
+  skipProps.push('customElementRef')
+
   return class extends React.Component {
     constructor (props) {
       super(props)
-      this.ref = (el) => (this.el = el)
+      this.ref = (el) => {
+        if (this.props.customElementRef) {
+          this.props.customElementRef.current = el
```

Comment:
Will fail if the user of the custom element uses an older [callback ref](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs) instead of `React.createRef()`

Reply:
Updated the PR.